<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>751</width>
    <height>527</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gesti√≥n de miEmpresa</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../Icons/logo.ico</normaloff>../Icons/logo.ico</iconset>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QMdiArea" name="mdiArea">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustIgnored</enum>
      </property>
      <property name="background">
       <brush brushstyle="NoBrush">
        <color alpha="255">
         <red>160</red>
         <green>160</green>
         <blue>160</blue>
        </color>
       </brush>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>751</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="opc_archivo">
    <property name="styleSheet">
     <string notr="true">QMenuBar::item {
    background-color: transparent;
}</string>
    </property>
    <property name="title">
     <string>Mantenimientos</string>
    </property>
    <addaction name="menuClientes"/>
    <addaction name="menuTipoClientes"/>
    <addaction name="separator"/>
    <addaction name="menuProductos"/>
    <addaction name="separator"/>
    <addaction name="menuSalir"/>
   </widget>
   <addaction name="opc_archivo"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="menuClientes">
   <property name="icon">
    <iconset>
     <normaloff>../Icons/clientes2.ico</normaloff>../Icons/clientes2.ico</iconset>
   </property>
   <property name="text">
    <string>Clientes</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+C</string>
   </property>
  </action>
  <action name="actionSalir">
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
  <action name="menuTipoClientes">
   <property name="icon">
    <iconset>
     <normaloff>../Icons/tipoClientes.ico</normaloff>../Icons/tipoClientes.ico</iconset>
   </property>
   <property name="text">
    <string>Tipo Clientes</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+T</string>
   </property>
  </action>
  <action name="menuSalir">
   <property name="icon">
    <iconset>
     <normaloff>../Icons/salir.ico</normaloff>../Icons/salir.ico</iconset>
   </property>
   <property name="text">
    <string>Salir</string>
   </property>
   <property name="iconText">
    <string>Salir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+Q</string>
   </property>
  </action>
  <action name="menuProductos">
   <property name="text">
    <string>Productos</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
